extensions/country-lookup/src/cache.ts

```diff
@@ -0,0 +1,48 @@
+import { environment, trash } from "@raycast/api";
+import { readFileSync } from "fs";
+import { writeFile } from "fs/promises";
+import { resolve } from "path";
+import { Cache, Key } from "swr";
+
+const CACHE_KEY = "swr-cache";
```

Comment:
Shouldn't the cache key be unique for the command here?

Reply:
There is no need to change anything here since it's stored inside the extension's own directory